REST/src/main/resources/static/js/board.js

```diff
@@ -1,9 +1,18 @@
 var myBoard = document.getElementById("myBoard");
 var opponentBoard = document.getElementById("opponentBoard");
+var timerIntervalID;
+var myRound = true;
+var stompClient = null;
 
 window.onload = function load() {
     setDivsForBoard(myBoard, false, true);
```

Comment:
weird trues and falses

Reply:
it's done already, will be shown in next PR